src/libs/antares/study/variable-print-info.h

```diff
@@ -132,27 +129,32 @@ class AllVariablesPrintInfo
         return numberSelectedLinkVariables;
     }
 
-private:
-    void setAllPrintStatusesTo(bool b);
     void computeMaxColumnsCountInReports();
+    void setAllPrintStatusesTo(bool b);
+    void reverseAll();
+
+    unsigned int numberOfEnabledVariables();
+    std::vector<std::string> namesOfEnabledVariables();
+    std::vector<std::string> namesOfDisabledVariables();
```

Comment:
Avoid copies (`std::string_view` are non-owning references to char ranges)
```suggestion
    std::vector<std::string_view> namesOfEnabledVariables();
    std::vector<std::string_view> namesOfDisabledVariables();
```

Reply:
The returned **std::vector<std::string**> is an **rvalue**, meaning a temporary value.
Do we really want to make it a **std::vector<std::string_view>** ?